initramfs/scripts/ubuntu-core-rootfs

```diff
@@ -132,6 +132,24 @@ handle_writable_paths()
 			continue
 		fi
 	done
+
+        # now apply the defaults on top
+        handle_writable_defaults
+}
+
+# handle_writable_default will apply system defaults *once* after
+# the handle_writable_path got populated
+handle_writable_defaults()
+{
+        srcpath="${rootmnt}/writable/system-data-defaults"
+        if [ ! -d "$srcpath" ] || [ -e "${srcpath}/.done" ]; then
+                return
+        fi
+
+        dstpath="${rootmnt}/writable/system-data"
+        cp -a "$srcpath/*" "$dstpath"
+        # XXX: or should we just delete the "system-data-defaults" dir here?
```

Comment:
It's nice to have it around to see what the defaults *were*. It's not tons of data, so perhaps it's fine to keep it around?

Reply:
The comment should be updated to reflect new name or removed alltogether.